addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -97,6 +97,10 @@ class StudyService extends Service {
     // For now, we assume that 'createdBy' and 'updatedBy' are always users and not groups
     const by = _.get(requestContext, 'principalIdentifier.uid');
 
+    if (rawData.category === 'Open Data' && by !== '_system_') {
```

Comment:
can you make a function named isSystem(), just like how we have isAdmin()?

Reply:
Done